### ISSUE TITLE
Minor issue BI lists

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#414]
We need to check this issue, inherited from BI, on our lists:
- https://github.com/italia/bootstrap-italia/issues/764

cc @astagi @OscarBresolin

**Comment by gpeirolo** [comment]
@Fupete any news about the lists?

**Comment by Fupete** [comment]
I just open a draft PR. Markup with simple `span`s. Still to correct the css handlers, give me an hour. Later I will ping Fabrizio for review, in the meantime, do you see any downside using a double nested `span`? In some way <em> is not correct ok, it is not an emphasys, but also a double span we lose hierarchy and without any . or : we will not have a pause in the screen reading experience... 🤔 

Eg. 
```
<span class="text">Main text
  <span class="description">Description text</span>
</span>
```

Instead of current:
```
<span class="text">Main text
  <em>Description text</em>
</span>
```

**Comment by Fupete** [comment]
Integrated also `_list.scss` with the correct `.description` handler, same PR. I also left the `em`'s one, just for retro-compatibility. Could be correct? 

In the meantime I asked Fabrizio a review, I will let you know @gpeirolo ✋

**Comment by gpeirolo** [comment]
@Fupete CSS code is correct. The span tag in the selector is not necessary but it's not a problem. 
Better leave the <em> for retro compatibility.

**Comment by Fupete** [comment]
Failed a11y review 👎 
I just left a comment on the issue on BI with the review.

**Comment by Fupete** [comment]
I change the status of this to TO DO lowering the priority to white. We will wait for the BI solution.
